rfcs/convergence/authoring-stories.md

```diff
@@ -373,26 +357,15 @@ export const ButtonWithIcon = () => <Button icon={<CalendarIcon />}>Text</Button
 
 ### 8. location and naming convention
 
-1. Story files should be in the same folder as the component which they are meant to show off. This will usually be the component which is the most unique to the story.
-2. Name of every .stories.tsx file should start with name of a component which it is targeting, to improve colocation in alphabetical file ordering. If appropriate, story name should be adjusted via configuration to a well readable name, suitable for documentation, like this:
-
-```tsx
-export const ButtonPrimary = (props: ButtonProps) => <Button {...props}>Text</Button>;
-ButtonPrimary.args = {
-  primary: true,
-};
-ButtonPrimary.storyName = 'Better story name';
-```
-
-3. If an internal story is broken out into its own file, then the file should mirror the name of the story, which will include the `Internal` keyword and a `.internal.stories` suffix for easier IDE search.
+All `.stories.tsx` files from one Storybook Docs page must be in a dedicated folder. If there is just one Docs page created, this should be `src/stories`. If there are multiple pages created, the folder should be `src/stories/page-name`.
```

Comment:
Calling out this line to make sure everybody is okay with it.

@ecraig12345 suggested that we could also enforce using `src/stories/page-name` folder even when there is a single page in the package. This would make it a bit easier to write code mods in the future, but the `.stories.tsx` extension might be sufficient for that.

After this gets resolved, I think this PR is ready to merge.